### DISCUSSION TITLE
TimeoutError: The write operation timed out

### DISCUSSION THREAD

**Discussion by formulatimer** [python-telegram-bot/python-telegram-bot#4073] [Q&A] [answered]
Hi,

I have written my code using version 13.7, and the bot sends photos properly. However, when I try to send a photo with a large size, it fails, and I am receiving the error "TimeoutError: The write operation timed out." I have increased the timeout, but I still received the same error. I am using urllib3 1.26.14.

I would like to change the version to v20. If I do that, will the problem be solved? Can you provide examples of how to adapt it? Which version of urllib3 do you recommend for me?

**Comment by Poolitzer** [comment, marked as answer]
You probably need to increase it more, but there is a max file size, have that in mind.

We did not change the default timeout in V20, but you can override it there as easily.
